builder/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:10
+
+ENV NODE_ENV=production
+
+RUN mkdir -p /home/node/builder
+WORKDIR /home/node/builder
+
+# Download docker cli
+RUN apt-get update && apt-get install -y -q \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
+    software-properties-common
+
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN apt-key fingerprint 0EBFCD88
+RUN add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/debian \
+   $(lsb_release -cs) \
+   stable"
+
+RUN apt-get update && apt-get install -y -q \
+    docker-ce-cli
+
+# Install builder dependencies
+COPY package.json .
+RUN npm install --only=production
+
+# Install builder
+COPY . .
+
+EXPOSE 8000
+
+# USER node
```

Comment:
Either uncomment or remove this line.

Reply:
It's used to run the container as user `node` instead of root user, that's why the app is located in`/home/node/app` rather than `/app/` because it's not a good practice to run a container as root user.

Reply:
I don't get why you removed it then.

(You can run the app in /app without being root)